examples/s3-import/locals.tf

```diff
@@ -4,11 +4,76 @@ locals {
   vpc_id                    = data.aws_vpc.supporting.id
   database_subnets          = local.database_subnet_id
   supporting_resources_name = "terraform-aws-rds"
```

Comment:
Can we use a variable instead or is this string required here for the provider/data source blocks?

Reply:
We can use a variable here for `supporting_resources_name` which is required by data sources